### PR TITLE
Migrations

### PR DESCRIPTION
**Pull request by PhilipDeFraties**
Summary:
Adds api directory with resources and models files for reports, adds migration for reports table
- [x] Updated Files: resources/reports.py, database/models.py, migrations/41059c651ae3.py
- [ ] Routes Added: GET /api/v1/reports, GET /api/v1/reports/1, DELETE /api/v1/reports/1, POST /api/v1/reports, PATCH /api/v1/reports/1
- [ ] Repo Representation: Front End, Back End, Amedaus, Open Weather, Google 
- [ ] Sad Path Tested
- [ ] Happy Path Tested
- [ ] All Tests are passing
- [ ] Updated README.md with added information
- [x] Colabotors: @Oxalisviolacea @Todd-Estes @evilaspaas1

### CODE REVIEW

File: api/__init__.py [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+            "success": False,
+            "error": 404,
+            "message": "resource not found"
+        }), 404
+
+    from api.resources.reports import ReportsResource, ReportResource
+
+    api.add_resource(ReportResource, '/api/v1/reports/<report_id>')
+    api.add_resource(ReportsResource, '/api/v1/reports')
+
```
Comment (Todd-Estes) [inline api/__init__.py:85]: 👍

File: api/resources/reports.py [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+
+from api import db
+from api.database.models import Report
+
+
+def _validate_field(data, field, proceed, errors, missing_okay=False):
+    if field in data:
+        # sanitize the report input here
+        data[field] = bleach.clean(data[field].strip())
+        if len(data[field]) == 0:
```
Comment (Todd-Estes) [inline api/resources/reports.py]: @PhilipDeFraties I think we ended up deleting line 15 and editing line 17 like so `if len(str(data[field])) == 0:`...python doesn't sanitize floats and won't check their lengths...so we just changed it to a string class IOT work...

**Review by Todd-Estes — APPROVED** [review]

File: api/database/models.py [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+    # lat
+    lat = Column(Float(13), unique=False, nullable=True)
+    # long
+    long = Column(Float(13), unique=False, nullable=True)
+    # description
+    description = Column(String(250), unique=False, nullable=True)
+    # event_type
+    event_type = Column(String(100), unique=False, nullable=True)
+    # image
+    image = Column(String(100), unique=False, nullable=True)
```
Comment (Oxalisviolacea) [inline api/database/models.py:25]: We can come back and fix this :)

**Review by Oxalisviolacea — APPROVED** [review]

File: api/resources/reports.py [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+from sqlalchemy.orm.exc import NoResultFound
+
+from api import db
+from api.database.models import Report
+
+
+def _validate_field(data, field, proceed, errors, missing_okay=False):
+    if field in data:
+        # sanitize the report input here
+        data[field] = bleach.clean(data[field].strip())
```
Comment (Oxalisviolacea) [inline api/resources/reports.py]: I think we need to remove the strip method here.

**Review by Oxalisviolacea — APPROVED** [review]

**Pull request merged**